packages/connect-popup/src/view/selectDevice.ts

```diff
@@ -41,10 +41,7 @@ const initWebUsbButton = (showLoader: boolean) => {
                 const channel = new BroadcastChannel(WEBEXTENSION.USB_PERMISSIONS_BROADCAST);
                 channel.onmessage = event => {
                     if (event.data.type === WEBEXTENSION.USB_PERMISSIONS_FINISHED) {
-                        postMessage({
-                            event: UI_EVENT,
-                            type: TRANSPORT.REQUEST_DEVICE,
-                        });
+                        postMessage({ type: TRANSPORT.REQUEST_DEVICE });
```

Comment:
From my previous work it seemed that `event` is necessary only in `CoreEventMessage`s and not in  `CoreRequestMessage`s (both of them together were previously `CoreMessage`). @martykan could you please quickly check that this fix of yours works without it? If not, my classification may be incorrect.

Reply:
Popup is not loading for me, trying to find the cause